modules/ipc/src/zenoh/service.cpp

```diff
@@ -75,6 +79,15 @@ auto callServiceRaw(Session& session, const TopicConfig& topic_config, std::span
 }
 
 auto getEndpointTypeInfoServiceTopic(const std::string& topic) -> std::string {
-  return fmt::format("type_info/{}", topic);
+  return fmt::format("{}/{}", TOPIC_INFO_SERVICE_TOPIC_PREFIX, topic);
+}
+
+auto isEndpointTypeInfoServiceTopic(const std::string& topic) -> bool {
+  std::vector<std::string> elements = absl::StrSplit(topic, '/');
+  if (elements.empty()) {
+    return false;
+  }
```

Comment:
you could use std::views::split for this

Reply:
nice!